core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -259,6 +259,8 @@
     
     public static final String SETSOFGENES_LOCATION = "querypage.setsofgenes.location";
 
+    public static final String WHOLE_SLIDE_VIEWER_SECRET_KEY = "viewer.key";
```

Comment:
`viewer.key` seems too generic. Maybe change it to "msk.whole.slide.viewer.secret.key". also change change the variable to `MSK_ WHOLE_SLIDE_VIEWER_SECRET_KEY`.